main.cpp

```diff
@@ -19,13 +19,28 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
+    CastCallBack(Rewriter& rewriter) : rewriter_(rewriter) {
+        
     };
 
     void run(const MatchFinder::MatchResult &Result) override {
-        // Your code goes here
+        if (const auto *F =
+                Result.Nodes.getNodeAs<clang::CStyleCastExpr>("cast")) {
+            const auto& SM = *Result.SourceManager;
+            const auto& BeginLoc = F->getLParenLoc();
+            const auto& EndLoc = F->getRParenLoc();
+            rewriter_.getEditBuffer(rewriter_.getSourceMgr().getMainFileID())
+                .ReplaceText(SM.getFileOffset(BeginLoc), 1, "static_cast<");
+            rewriter_.getEditBuffer(rewriter_.getSourceMgr().getMainFileID())
+                .ReplaceText(SM.getFileOffset(EndLoc), 1, ">");
```

Comment:
BTW, you can shorten the code:  
```suggestion
            rewriter_.ReplaceText(BeginLoc, 1, "static_cast<");
            rewriter_.ReplaceText(EndLoc, 1, ">");
```